### PR TITLE
[TDV-629] Add getRawResponse method for API caller

### PR DESCRIPTION
**Pull request by ardier16**
+ networkDetails getter

### CODE REVIEW

File: src/api2/api-caller.js [hunk 1]
```diff
@@ -70,6 +76,15 @@ export class ApiCaller {
     })
   }
 
+  getRawResponse (endpoint, query) {
```
Comment (max-nes) [inline src/api2/api-caller.js]: `getRaw`

File: src/api2/api-caller.js [hunk 2]
```diff
@@ -211,13 +227,17 @@ export class ApiCaller {
...
     }
 
-    response = middlewares.parseJsonapiResponse(response)
-
-    if (!isEmpty(response.links)) {
-      if (opts.needSign) {
-        response.makeLinkCallersWithSignature(this)
-      } else {
-        response.makeLinkCallers(this)
+    if (opts.needRawResponse) {
```
Comment (max-nes) [inline src/api2/api-caller.js]: `false` is default, let’s invert the condition and bodies

File: src/api2/api-caller.js [hunk 3]
```diff
@@ -175,6 +190,7 @@ export class ApiCaller {
    * @param {object} opts.query - request query params. See {@link parseQuery} for details
    * @param {string} opts.method - the http method of request
    * @param {bool} opts.needSign - defines if will try to sign the request, `false` by default
+   * @param {bool} opts.needRawResponse - defines if raw response should be returned, `false` by default
```
Comment (max-nes) [inline src/api2/api-caller.js]: `needRaw`

File: src/api2/api-caller.js (hunk 1 above)
Comment (ardier16) [reply src/api2/api-caller.js]: Renamed

File: src/api2/api-caller.js (hunk 2 above)
Comment (ardier16) [reply src/api2/api-caller.js]: Inverted

File: src/api2/api-caller.js (hunk 3 above)
Comment (ardier16) [reply src/api2/api-caller.js]: Renamed

**Pull request merged**